src/coreclr/nativeaot/Runtime/thread.h

```diff
@@ -27,7 +27,7 @@ class Thread;
 # endif
 #endif // HOST_64BIT
 
-#define TOP_OF_STACK_MARKER ((PTR_VOID)(uintptr_t)(intptr_t)-1)
+static PInvokeTransitionFrame* const TOP_OF_STACK_MARKER = (PInvokeTransitionFrame*)ptrdiff_t(-1);
```

Comment:
Initialized statics in headers tend to produce code bloat. Every compilation unit gets its own copy.

Reply:
I'd hope they can just elide it as a constant expression, but it can be a define.

Reply:
replaced with a define